File: abc143/abc143_c.cs
// https://atcoder.jp/contests/abc143/submissions/8107540

using System;

namespace CSharp
{
    class Program
    {
        public static void Main()
        {
            var N = int.Parse(Console.ReadLine());
            var S = Console.ReadLine();

            var d = 0;
            for (int i = 0; i < N - 1; i++)
            {
                if (S[i] == S[i + 1])
                {
                    d++;
                }
            }
            Console.WriteLine(N - d);
        }
    }
}

File: abc/abc143/abc143_c.fs
// https://atcoder.jp/contests/abc143/submissions/12054420

let N = stdin.ReadLine() |> int
let S = stdin.ReadLine().ToCharArray()

S
|> Array.pairwise
|> Array.filter (fun (x, y) -> x = y)
|> Array.length
|> fun d -> N - d
|> stdout.WriteLine
